src/main/cpp/src/api/genomicsdb.cc

```diff
@@ -237,15 +236,30 @@ GenomicsDBVariants GenomicsDB::query_variants(const std::string& array,
 
   query_config.validate();
 
-  return GenomicsDBVariants(TO_GENOMICSDB_VARIANT_VECTOR(query_variants(array, &query_config)),
-                            create_genomic_field_types(query_config, m_annotation_service));
+  // ALT pointer will be to the first std::string in a std::vector owned by an instance of VariantFieldALTData
+  auto field_types = create_genomic_field_types(query_config, m_annotation_service);
+
+  auto it = field_types.find("ALT");
+  if(it != field_types.end()) {
+    it->second.type_idx = genomic_field_type_t::genomicsdb_basic_type::STRING;
+  }
```

Comment:
Sorry to flip-flop...I'm noticing there are actually 4 places where this little blurb is repeated (create_genomics_field_types...find("ALT")...genomic_field_type_t::genomicsdb_basic_type::STRING). Can you pull that into a small function so we don't repeat it in so many different places?